Resources/doc/index.md

```diff
@@ -120,6 +120,24 @@ snc_redis:
 Please note that the master dsn connection needs to be tagged with the ```master``` alias.
 If not, `predis` will complain.
 
+A setup using `predis` sentinel replication could look like this:
+
+``` yaml
+snc_redis:
+    clients:
+        default:
+            type: predis
+            alias: default
+            dsn:
+                - redis://localhost
```

Comment:
Are these sentinel hosts or redis hosts? What about port? Does the `replication: sentinel` setting automatically use port `26379` on those hostnames?